ArduCopter/events.cpp

```diff
@@ -322,7 +323,9 @@ void Copter::failsafe_deadreckon_check()
     const char* dr_prefix_str = "Dead Reckoning";
 
     // get EKF filter status
-    bool ekf_dead_reckoning = inertial_nav.get_filter_status().flags.dead_reckoning;
+    nav_filter_status filt_status {};
+    AP::ahrs().get_filter_status(filt_status);
+    bool ekf_dead_reckoning = filt_status.flags.dead_reckoning;
```

Comment:
```suggestion
    const bool ekf_dead_reckoning = AP::ahrs().has_status(SomeNamespace::DEAD_RECKONING);
```

Reply:
What about being able to do `AP::ahrs().has_status(SomeNamespace::HORIZ_VEL | SomeNamespace::VERT_VEL)` to check that both horizontal and vertical available.